src/site/stages/build/process-cms-exports/schemas/transformed/node-person_profile.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-person_profile'] },
+    entityType: { enum: ['node'] },
+    entityBundle: { enum: ['person_profile'] },
+    entityUrl: {
+      // This one doesn't have a breadcrumb, interestingly enough
+      type: 'object',
+      properties: {
+        path: { type: ['string', 'null'] },
+      },
+      required: ['path'],
+    },
+    fieldBody: { type: ['string', 'null'] },
+    fieldDescription: { type: ['string', 'null'] },
+    fieldEmailAddress: { type: ['string', 'null'] },
+    fieldLastName: { type: 'string' },
+    fieldMedia: { type: ['object', 'null'] },
+    fieldNameFirst: { type: 'string' },
+    fieldOffice: {
+      type: 'object',
```

Comment:
This isn't a ref to `transformed/field-office` because we only select a few of the properties from the office for this entity.